PubSub/src/Connection/Grpc.php

```diff
@@ -106,6 +107,13 @@ public function __construct(array $config = [])
      */
     public function createTopic(array $args)
     {
+        if (isset($args['messageStoragePolicy'])) {
+            $args['messageStoragePolicy'] = $this->serializer->decodeMessage(
```

Comment:
Can we get away with `new MessageStoragePolicy($args['messageStoragePolicy']);`?

Reply:
No, we'd need to manually convert the casing of the argument, `allowedPersistenceRegions`, and it would be difficult to maintain forward-compatibility of further arguments were added to the `MessageStoragePolicy` message.

Reply:
WDYT about the generated protobuf messages accepting both camel and snake case keys? It's too bad we have to keep using the serializer when we're so close to being able to use the native functionality.